pom.xml

```diff
@@ -932,6 +932,12 @@
         <groupId>com.smoketurner</groupId>
         <artifactId>dropwizard-swagger</artifactId>
         <version>${swagger.version}</version>
+        <exclusions>
```

Comment:
can you add a comment to give the source of the conflict @shounakmk219 ?

Reply:
Done